metrics_api/lib/opentelemetry/metrics/instrument/asynchronous_instrument.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Metrics
+    module Instrument
+      # https://opentelemetry.io/docs/reference/specification/metrics/api/#asynchronous-instrument-api
+      class AsynchronousInstrument
+        attr_reader :name, :unit, :description, :callback
+
+        # @api private
+        def initialize(name, unit: nil, description: nil, callback: nil)
+          @name = name
+          @unit = unit || ''
+          @description = description || ''
+          @callback = callback ? Array(callback) : []
```

Comment:
nit: since this is an array of functions, can we pluralize the name as `callbacks`?

Reply:
Sure, I can pluralize them!

I think the reason I initially didn't do that was to keep the same "symbol same" from the specification although this `@callback` instance variable is internal.

But in `Meter` API, in `metrics_api/lib/opentelemetry/metrics/meter.rb`, should I pluralize the keyword arg `callback:` as well in the `create_observable_counter`, `create_observable_gauge`, and `create_observable_up_down_counter` methods?